src/modules/core/components/map/map.module.scss

```diff
@@ -0,0 +1,99 @@
+@use '@/styles/variables.module';
+
+.map_container {
+  position: relative;
+  height: 500px;
+  display: flex;
+}
+
+// marker with photo
+.custom_icon_container {
```

Comment:
name all css classes in camelCase style - `customIconContainer`. And mayble remove `custom_` prefix in the class name. So, just `iconContainer`, `avatar` and etc...